### PR TITLE
Added Mark-device-as... patch to 1.20 and minor changes to other patches

### LINKED ISSUE #1072: Check for patch changes for 1.20-18

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

closes #1072 

Part of 1.20-18 release in https://github.com/aws/eks-distro/issues/1067

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/kubernetes/1-20/patches/0022-EKS-PATCH-Update-aws-sdk-to-v1.42.23-for-ap-se-3.patch [hunk 1]
```diff
@@ -2,9 +2,9 @@ From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Jyoti Mahapatra <jyotima@amazon.com>
 Date: Thu, 31 Mar 2022 18:28:59 -0700
 Subject: --EKS-PATCH-- Update aws-sdk to v1.42.23 for ap-southeast-3
- region build
 
 Update aws-sdk to v1.42.23 for ap-southeast-3 region build
+
```
Comment (kevinsalerno) [inline projects/kubernetes/kubernetes/1-20/patches/0022-EKS-PATCH-Update-aws-sdk-to-v1.42.23-for-ap-se-3.patch:7]: Was this whitespace intentional??

Comment (kschumy) [reply projects/kubernetes/kubernetes/1-20/patches/0022-EKS-PATCH-Update-aws-sdk-to-v1.42.23-for-ap-se-3.patch:7]: Yeah it was. I probably should have clarified this in the description, but this is a change needed for the automation Bhavitha is working on

**Pull request merged**